crates/uv/src/settings.rs

```diff
@@ -2551,23 +2491,9 @@ pub(crate) struct ResolverInstallerSettingsRef<'a> {
 #[allow(clippy::struct_excessive_bools)]
 #[derive(Debug, Clone, Default)]
 pub(crate) struct ResolverInstallerSettings {
-    pub(crate) index_locations: IndexLocations,
-    pub(crate) index_strategy: IndexStrategy,
-    pub(crate) keyring_provider: KeyringProviderType,
-    pub(crate) resolution: ResolutionMode,
-    pub(crate) prerelease: PrereleaseMode,
-    pub(crate) fork_strategy: ForkStrategy,
-    pub(crate) dependency_metadata: DependencyMetadata,
-    pub(crate) config_setting: ConfigSettings,
-    pub(crate) no_build_isolation: bool,
-    pub(crate) no_build_isolation_package: Vec<PackageName>,
-    pub(crate) exclude_newer: Option<ExcludeNewer>,
-    pub(crate) link_mode: LinkMode,
+    pub(crate) resolver_settings: ResolverSettings,
```

Comment:
I'd prefer to call this `resolver`, what do you think?

Reply:
`resolver_settings` is definitely clunky. I went back and forth but I've updated it to `resolver` now.